map_info.yaml

```diff
@@ -0,0 +1,24 @@
+#where the map information shall live.
+---
+player:
+  column: 1
+  row: 1
+goblin_one:
```

Comment:
Your metadata is a bit insufficient and it's kind of causing you to have to know too much about what you are pulling when you parse. You should have a "monster" tag and in that tag should be data like the position of the monster and it's key in your monster data yml (you should have one if you don't). This is why you are stuck specifically trying to load goblin one and goblin two and such in your code.